numpy/core/code_generators/generate_umath.py

```diff
@@ -64,6 +67,15 @@ def __init__(self, type, f=None, in_=None, out=None, astype=None, simd=None):
             out = out.replace('P', type)
         self.out = out
         self.simd = simd
+        """
+        TODO:
+           - remove 'SIMD' after integrate the rest of simd loops
```

Comment:
@seiko2plus I'm sorry, I mis-understood. I did not realize this was a TODO for the future. I thought you already for rid of the simd. So please ignore the question I asked towards the end of the meeting.